airbyte-config/models/build.gradle

```diff
@@ -27,3 +24,30 @@ jsonSchema2Pojo {
     includeSetters = true
     serializable = true
 }
+
+test {
+    useJUnitPlatform {
+        excludeTags 'log4j2-config', 'logger-client'
```

Comment:
Is it possible to use an `integration-test` tag instead so we don't have to add individual tags here? Then it'll also be more clear when reading the code why the tag is there.

I kind of like this pattern more than our current `integrationTest` setup that involves separate modules.

Reply:
i think the argument for separate modules is that sometimes the deps are pretty different. normalization integration tests are a good example of this. though you could probably argue that what we are doing there is an antipattern in terms of project structure.

Reply:
I also favor this set up because I think it's much easier on gradle as well. We could defer to this approach for everything except for base normalisation?

the rest I split these tags are that we want the `logger-client` tests to run once, and the `log4j2-config` tests to run thrice. I looked into this and invalidation happens at the task level so combining the tags would result in them running 3 times. It's a small performance penalty (single digit minutes) so happy to do so if people think that's more simple.